setup.py

```diff
@@ -61,6 +64,12 @@
     )
     ext_modules.append(ext)
 
+# Hack to install ta-lib library loading into virtualenv
+def install_virtualenv_lib_loader():
+    with open(os.getenv('VIRTUAL_ENV', './venv') + '/bin/activate', 'a') as f:
```

Comment:
Hm, maybe it would be great if the Python package would also include (or download) and compile the `ta-lib` library if it is not installed system-wide? Like some packages do?